templates/product_template.xml

```diff
@@ -9,15 +9,15 @@
         <xpath expr="//*[@id='product_details']">
             <t t-if="product.website_attachment_ids">
                 <hr />
-                <div class="card o_not_editable" id="product_attachments">
```

Comment:
@MaganaAsiati 
You don't "implement the functionnality", it's supposed to be there but it doesn't work anymore. 
So you fix it. Not an [IMP], a [FIX] + gse_product and not GSE_PRODUCT
Also, you can add a .gitignore to prevent the .pyc files.